docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Runtime package store | Microsoft Docs
+description: This topic explains the runtime package store and target manifests used by .NET Core.
+keywords: .NET, .NET Core, dotnet store, runtime package store
+author: bleroy
```

Comment:
that doesn't take away he's authorship @Rick-Anderson. I'm listed as the ms.author